skaffold.yaml

```diff
@@ -61,6 +61,7 @@ portForward:
   namespace: default
   port: 80
   localPort: 8000
+  address: 0.0.0.0
```

Comment:
isn't this something that should be done on multipass/VM setup?
reason is this might clash with something else running on the machine if you are using the main host compared to a VM

Reply:
No worries, I'll revert this change. It would be nice if this could be set via an environment variable or flag, but I can't find an obvious way that skaffold supports this.